src/app/file/gif_format.cpp

```diff
@@ -493,9 +494,12 @@ class GifDecoder {
   }
 
   // Adds colors used in the GIF frame so we can draw it over
-  // m_currentImage. If the frame contains a local colormap, we try to
-  // find them in the current sprite palette (using
-  // Palette::findExactMatch()) so we don't add duplicated entries.
+  // m_currentImage. If the frame 0 contains global palette and no local palette
+  // we'll take the entire global palette as a pelette, no matter if it's
+  // used or not in the image.
+  // On the other hand if the frame contains local palette, the colors will be
+  // added to the palette of the sprite via Palette::findExactMatch()) so
+  // we don't add duplicated entries.
```

Comment:
You can reword/spellcheck this comment (and I think frames doesn't contain the global colormap, but the file is the one with the global palette, frames contain local colormaps).